beacon-chain/state/state-native/v1/state_trie.go

```diff
@@ -195,12 +234,34 @@ func (b *BeaconState) HashTreeRoot(ctx context.Context) ([32]byte, error) {
 
 	b.lock.Lock()
 	defer b.lock.Unlock()
+
 	if err := b.initializeMerkleLayers(ctx); err != nil {
 		return [32]byte{}, err
 	}
+
+	if b.merkleLayers == nil || len(b.merkleLayers) == 0 {
```

Comment:
so you are basically redoing what the above method does :) , please remove this chunk.

Reply:
This is odd... It could be a result of some merge commit